src/DotNetOutdated/Program.cs

```diff
@@ -108,6 +108,10 @@ internal class Program : CommandBase
         [Option(CommandOptionType.NoValue, Description = "Treat package source failures as warnings.", ShortName = "ifs", LongName = "ignore-failed-sources")]
         public bool IgnoreFailedSources { get; set; } = false;
 
+        [Option(CommandOptionType.NoValue, Description = "Include all dependencies in the report even the ones not outdated",
+            ShortName = "a", LongName = "all")]
+        public bool All { get; set; } = false;

```

Comment:
```suggestion
        [Option(CommandOptionType.NoValue, Description = "Include all dependencies in the report even the ones not outdated",
            ShortName = "utd", LongName = "inlcude-up-to-date")]
        public bool IncludeUpToDate { get; set; } = false;

```

This PR looks great, my only suggestion is to change the name to be a little more obvious

Reply:
@slang25 updated the PR as per your review. Also updated the README to include the new argument